src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((base1 + base2) / 2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid, area: "
+                + getArea() + " sq. units, base1: "
+                + base1 + " units, base2: "
+                + base2 + " units, height: "
+                + height + " units, color: "
+                + getColor());
```

Comment:
Instead of directly printing to the console within the `draw` method, consider using a method that returns the string representation of the object for better testability and separation of concerns.